### PR TITLE
Update GTK install docs

### PR DESCRIPTION
**Pull request by vadz**
Reformat, remove obsolete parts, add prerequisites section (@swt2c please let me know if I wrote anything stupid about RedHat systems).

### CODE REVIEW

File: docs/gtk/install.md [hunk 1]
```diff
@@ -13,38 +13,66 @@ The simplest case                      {#gtk_simple}
...
+
+wxMediaCtrl implementation requires GStreamer, i.e. either
+`libgstreamerVERSION-dev` or `gstream-VERSION-devel`, where `VERSION` can be
+either 0.10 or 1.0 (recommended).
+
+wxWebRequest requites libcurl development package, e.g. `libcurl4-openssl-dev`
+or `libcurl-devel`.
+
+wxWebView requires `libwebkit2gtk-4.0-dev` or `libwebkitgtk-3.0-dev` under
+Debian and `webkit2gtk-devel` under Fedora.
```
Comment (swt2c) [inline docs/gtk/install.md:61]: Fedora should be `webkit2gtk3-devel`.  Also you can probably drop `libwebkitgtk-3.0-dev` from the Debian list as I'm pretty sure that version has been removed (as it's not longer supported).

File: docs/gtk/install.md [hunk 2]
```diff
@@ -13,38 +13,66 @@ The simplest case                      {#gtk_simple}
...
+
+At the very least, you will need GTK libraries themselves. Unfortunately the
+exact package names differ between different Linux distributions, but for
+Debian and Debian-based distribution these libraries are part of `libgtk-3-dev`
+package, while in Fedora and other RPM-based distributions the same package is
+known as `gtk3-devel`.
+
+For OpenGL support, you need `libgl1-mesa-dev` and `libglu1-mesa-dev` packages
+under Debian and `mesa-libGL-devel` and `mesa-libGLU-deve` under Fedora. For
+EGL support, `libegl1-mesa-dev` or `mesa-libEGL-devel` is needed.
```
Comment (swt2c) [inline docs/gtk/install.md:51]: Missing the `l` at the end of `mesa-libGLU-devel`.

File: docs/gtk/install.md [hunk 3]
```diff
@@ -13,38 +13,66 @@ The simplest case                      {#gtk_simple}
...
+
+wxMediaCtrl implementation requires GStreamer, i.e. either
+`libgstreamerVERSION-dev` or `gstream-VERSION-devel`, where `VERSION` can be
+either 0.10 or 1.0 (recommended).
```
Comment (swt2c) [inline docs/gtk/install.md:55]: On Fedora, the package name is `gstreamer1-devel` but for some reason we install `gstreamer1-plugins-base-devel`.  Can't remember the reasoning for the plugins-base package.  Similarly, on Debian we install `libgstreamer-plugins-base1.0-dev`.

File: docs/gtk/install.md (hunk 1 above)
Comment (vadz) [reply docs/gtk/install.md:61]: Fixed, thanks!

File: docs/gtk/install.md (hunk 3 above)
Comment (vadz) [reply docs/gtk/install.md:55]: Thanks for noticing this, we indeed need `libgstreamer-plugins-base1.0-dev` as some headers we use are contained in it, and not `libgstreamer1.0-dev` itself (which is its dependency).

**Pull request merged**